README.md

```diff
@@ -1,4 +1,6 @@
-# Dapr pub/sub
+# Microservice communication using pubsub (async)
+
+![](images/pubsub-diagram.png)
```

Comment:
Love it directionally. Can we :
- show service bus instead of redis cache?
- hint at checkout and order processor being container apps?
- show a container app style URL?

Reply:
Done. Please let me know what you think and if it looks good, I can port to the other language examples.